website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx

```diff
@@ -44,72 +44,68 @@ The POST request submits a JSON object with the following properties as a reques
+
+#### Post-Plan and Pre-Apply Properties
+
+Requests with `stage` set to `post_plan` or `pre_apply` contain the following additional properties.
+
+|      Key path       |  Type  | Values |                        Description                        |
+| ------------------- | ------ | ------ | --------------------------------------------------------- |
 | `plan_json_api_url` | string |        | The URL to retrieve the JSON Terraform plan for this run. |
 
 ### Sample Payload
 
 ```json
 {
   "payload_version": 1,
-  "access_token": "4QEuyyxug1f2rw.atlasv1.iDyxqhXGVZ0ykes53YdQyHyYtFOrdAWNBxcVUgWvzb64NFHjcquu8gJMEdUwoSLRu4Q",
   "stage": "post_plan",
+  "access_token": "4QEuyyxug1f2rw.atlasv1.iDyxqhXGVZ0ykes53YdQyHyYtFOrdAWNBxcVUgWvzb64NFHjcquu8gJMEdUwoSLRu4Q",
+  "configuration_version_download_url": "https://app.terraform.io/api/v2/configuration-versions/cv-ntv3HbhJqvFzamy7/download",
+  "configuration_version_id": "cv-ntv3HbhJqvFzamy7",
   "is_speculative": false,
-  "task_result_id": "taskrs-2nH5dncYoXaMVQmJ",
-  "task_result_enforcement_level": "mandatory",
-  "task_result_callback_url": "https://app.terraform.io/api/v2/task-results/5ea8d46c-2ceb-42cd-83f2-82e54697bddd/callback",
+  "organization_name": "hashicorp",
+  "plan_json_api_url": "https://app.terraform.io/api/v2/plans/plan-6AFmRJW1PFJ7qbAh/json-output",
   "run_app_url": "https://app.terraform.io/app/hashicorp/my-workspace/runs/run-i3Df5to9ELvibKpQ",
-  "run_id": "run-i3Df5to9ELvibKpQ",
-  "run_message": "Triggered via UI",
   "run_created_at": "2021-09-02T14:47:13.036Z",
   "run_created_by": "username",
-  "workspace_id": "ws-ck4G5bb1Yei5szRh",
-  "workspace_name": "tfr_github_0",
-  "workspace_app_url": "https://app.terraform.io/app/hashicorp/my-workspace",
-  "organization_name": "hashicorp",
-  "plan_json_api_url": "https://app.terraform.io/api/v2/plans/plan-6AFmRJW1PFJ7qbAh/json-output",
-  "vcs_repo_url": "https://github.com/hashicorp/terraform-random",
+  "run_id": "run-i3Df5to9ELvibKpQ",
+  "run_message": "Triggered via UI",
+  "task_result_callback_url": "https://app.terraform.io/api/v2/task-results/5ea8d46c-2ceb-42cd-83f2-82e54697bddd/callback",
+  "task_result_enforcement_level": "mandatory",
+  "task_result_id": "taskrs-2nH5dncYoXaMVQmJ",
   "vcs_branch": "main",
+  "vcs_commit_url": "https://github.com/hashicorp/terraform-random/commit/7d8fb2a2d601edebdb7a59ad2088a96673637d22",
   "vcs_pull_request_url": null,
-  "vcs_commit_url": "https://github.com/hashicorp/terraform-random/commit/7d8fb2a2d601edebdb7a59ad2088a96673637d22"
+  "vcs_repo_url": "https://github.com/hashicorp/terraform-random",
+  "workspace_app_url": "https://app.terraform.io/app/hashicorp/my-workspace",
+  "workspace_id": "ws-ck4G5bb1Yei5szRh",
+  "workspace_name": "tfr_github_0",
+  "workspace_working_directory": "/working_directory"
```

Comment:
If folks don't actually name their working directories "/working_directory", could we offer a more descriptive example name? Like "production" or "test"?

Reply:
Changed to `/terraform`.  I looked at samples in TFC and this was a common working dir.